types/node/globals.d.ts

```diff
@@ -626,17 +635,17 @@ declare namespace NodeJS {
         isPaused(): boolean;
         pipe<T extends WritableStream>(destination: T, options?: { end?: boolean; }): T;
         unpipe(destination?: WritableStream): this;
-        unshift(chunk: string | Buffer | Uint8Array, encoding?: BufferEncoding): void;
```

Comment:
Isn't `encoding` part of a readable stream? https://nodejs.org/dist/latest-v12.x/docs/api/stream.html#stream_readable_unshift_chunk_encoding